### DISCUSSION TITLE
Taxonomy issue

### DISCUSSION THREAD

**Discussion by AdamSmallman** [statamic/cms#10367] [Troubleshooting]
Hello im using Taxonomy on a collection called "update-posts" and in that "update-posts" folder I have created a "tags" folder with both an index and show .antlers.html. But when I click on one of the tags on the website I get a 404 page. 

I followed the following tutorial: https://laracasts.com/series/learn-statamic-with-jack/episodes/7

Any ideas? 

![Screenshot 2024-06-25 at 11 24 14 2](https://github.com/statamic/cms/assets/17714274/46c5257e-9aca-4b01-9bae-ec027183a6ce)

<img width="1512" alt="Screenshot 2024-06-25 at 11 25 21" src="https://github.com/statamic/cms/assets/17714274/b6b10f5d-9607-4eb4-8f74-fbd93e1e2726">

<img width="1104" alt="Screenshot 2024-06-26 at 09 23 00" src="https://github.com/statamic/cms/assets/17714274/aab387b4-db0b-413d-861f-028ee2b96926">

**Comment by edalzell** [comment]
is that collection wired up to the `tags` taxonomy? Let's see the collection config

**Reply by AdamSmallman** [reply]
<img width="1512" alt="Screenshot 2024-06-26 at 21 35 01" src="https://github.com/statamic/cms/assets/17714274/895baac9-2f11-4f8b-abb2-0219a1eeb2ff">

Whats strange is that http://e164-website.test/updates/update-tags/test shows the 404 page not found but if I delete updates the test tag page shows e.g: http://e164-website.test/update-tags/test

I started fresh by doing the following steps:

- Created a page collection called **updates**  
- Created a collection called **update** (For the "articles/posts" )
- I then Mount the update collection to updates 
- Next I created the Taxonomies and attached it to updates

**Reply by edalzell** [reply]
and what is the handle of the taxonomy?

**Reply by AdamSmallman** [reply]
Update tags so it will be **update_tags**

<img width="949" alt="Screenshot 2024-06-26 at 21 55 55" src="https://github.com/statamic/cms/assets/17714274/ce660c9d-11e6-4143-9dc7-59f189fe1cf7">

**Reply by AdamSmallman** [reply]
<img width="283" alt="Screenshot 2024-06-26 at 22 04 00" src="https://github.com/statamic/cms/assets/17714274/0122a408-1c4d-4eb2-bfc3-257aa2adb935">

**Reply by AdamSmallman** [reply]
This URL works: **http://e164-website.test/update-tags** and this doesn't **http://e164-website.test/update_tags**
